### PR TITLE
#6 Implement and test termination (by evaluation steps count

### PR DESCRIPTION
**Pull request by karolmie1**
Yup, I know termination by closeness to maximum is still missing, but for termination by number of steps will have to suffice.

### CODE REVIEW

File: alhe/alhe.Rproj [hunk 1]
```diff
@@ -18,3 +18,4 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
+PackageRoxygenize: rd,collate,namespace
```
Comment (mklecha) [inline alhe/alhe.Rproj]: No idea what it does

File: alhe/tests/testthat/testGenerateStartPoints.R [hunk 2]
```diff
@@ -6,3 +6,13 @@ test_that("Generating start points works", {
   result = rep(list(temp), popCount);
   expect_that(x, equals(result));
 })
+
+test_that("TerminationByEvaluationsCount return proper values depending on different history length", {
+  popCount = 3;
+  notEnoughEvaluations = popCount - 1;
+  enoughEvaluations = popCount + 4;
```
Comment (mklecha) [inline alhe/tests/testthat/testGenerateStartPoints.R]: Shouldn't we test it on the border values? (popCount, popCount+1 etc?)

**Comment by mklecha** [comment]
I'd like you to look closely at the tests if you don't mind.

File: alhe/tests/testthat/testGenerateStartPoints.R (hunk 2 above)
Comment (karolmie1) [inline alhe/tests/testthat/testGenerateStartPoints.R]: yep, why not. It doesn't matter much, as it is just basic test does it work or not, but if we want to be thorougth then why not.

File: alhe/alhe.Rproj (hunk 1 above)
Comment (karolmie1) [inline alhe/alhe.Rproj]: @mklecha me too it was autogenerated by documentation tool, so I guess it is a good change :)

**Pull request merged**